src/components/CharactersPage/CharacterForm/CharacterForm.js

```diff
@@ -62,7 +64,27 @@ const CharacterForm = ({ onClose }) => {
     return params[0] === formModeEnum.edit
   }, [params])
 
-  const character = useSelector(characterSelectors.character(params[1]))
+  const characterCacheId = client.cache.identify({
+    __typename: "Person",
+    id: params[1],
+  })
+
+  const {
+    error: getACharacterError,
+    loading: getACharacterLoading,
+    data: character,
+  } = useQuery(GET_A_CHARACTER, {
+    fetchPolicy: "cache-first",
```

Comment:
This is the default fetch policy, there is no need to explicitly specify it like this